test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcCSharpRazorCodeParserTest.cs

```diff
@@ -273,6 +273,50 @@ public void ParseModelKeyword_ErrorOnInheritsFollowedByModel()
         }
 
         [Theory]
+        [InlineData("IMyService Service;", "IMyService", "Service")]
+        [InlineData("IMyService Service;;", "IMyService", "Service")]
+        [InlineData("  Microsoft.AspNet.Mvc.IHtmlHelper<MyNullableModel[]?>  MyHelper;  ",
+                    "Microsoft.AspNet.Mvc.IHtmlHelper<MyNullableModel[]?>", "MyHelper")]
+        [InlineData("  Microsoft.AspNet.Mvc.IHtmlHelper<MyNullableModel[]?>  MyHelper;  ",
+                    "Microsoft.AspNet.Mvc.IHtmlHelper<MyNullableModel[]?>", "MyHelper")]
```

Comment:
this data looks identical to the one above.  what am I missing?  if nothing, cleanup to avoid creating duplicate tests